### PR TITLE
refactor(package): remove dublicates build, serve

### PR DESCRIPTION
**Pull request by Kaskyi**
refactor(package): remove dublicates build, serve, dev

### CODE REVIEW

**Review by sergeysenja1992 — CHANGES_REQUESTED** [review]
we need clarify number of commands that we using BEFORE remove it

File: package.json [hunk 1]
```diff
@@ -25,21 +25,10 @@
     "xm-ngx": "xm-ngx",
     "postinstall": "node ./scripts/build-packages.js && ngcc --properties esm5 browser module main",
     "prebuild": "xm-ngx ext-lazy-module && xm-ngx ext-assets && xm-ngx ext-i18n && xm-ngx ext-theming && xm-ngx ext-themes && xm-ngx dynamic-specification && npm run import-jsf-exts",
-    "start": "npm run prebuild && ng serve",
-    "start:prod": "npm run prebuild && ng serve --configuration=serve-production",
-    "start-local": "npm run prebuild && ng serve --proxy-config local.proxy.conf.js --live-reload",
-    "start-local:aot": " npm run prebuild && ng serve --proxy-config local.proxy.conf.js --live-reload  --host=0.0.0.0 --configuration production --aot=true",
-    "dev": "npm run prebuild && ng serve --proxy-config local.proxy.conf.js --live-reload --host=0.0.0.0",
-    "dev-idp": "npm run prebuild && ng serve --proxy-config local.proxy.conf.js --live-reload --host=0.0.0.0 --disable-host-check  -c idp",
-    "dev:aot": "npm run prebuild && ng serve --proxy-config local.proxy.conf.js --live-reload  --host=0.0.0.0 --configuration production --aot=true",
```
Comment (sergeysenja1992) [inline package.json:34]: how to start local with aot?

Comment (Sumragen) [reply package.json:34]: AOT is the default compiler starting in Angular 9.
Moreover, according to `angular.json` config file `serve-production` configuration includes `aot: true`.

**Review by Sumragen — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -25,21 +25,10 @@
...
+    "start:prod": "npm run prebuild && ng serve --proxy-config local.proxy.conf.js --live-reload --host=0.0.0.0 --configuration serve-production",
     "build": "ng build --stats-json",
-    "build:prod": "npm run prebuild && ng build --configuration production",
-    "build-aot": "npm run prebuild && ng build --configuration production --aot=true",
-    "build-no-aot": "npm run prebuild && ng build --configuration production --aot=false",
-    "build-prod-v1": "npm run prebuild && ng build -pr --configuration production",
-    "build-prod-v1-stat": "npm run prebuild && ng build -pr --configuration production --statsJson",
-    "build-prod-v2": "npm run prebuild && ng build --configuration production --aot=true --build-optimizer=true --vendor-chunk=true --common-chunk=true",
-    "build-prod-v2-stat": "npm run prebuild && ng build --configuration production --aot=true --build-optimizer=true --vendor-chunk=true --common-chunk=true --statsJson",
+    "build:prod": "npm run prebuild && ng build --configuration production --statsJson",
```
Comment (sergeysenja1992) [inline package.json:31]: @Kaskyi  we need build-aot alias

File: package.json [hunk 3]
```diff
@@ -25,21 +25,10 @@
...
     "postinstall": "node ./scripts/build-packages.js && ngcc --properties esm5 browser module main",
     "prebuild": "xm-ngx ext-lazy-module && xm-ngx ext-assets && xm-ngx ext-i18n && xm-ngx ext-theming && xm-ngx ext-themes && xm-ngx dynamic-specification && npm run import-jsf-exts",
-    "start": "npm run prebuild && ng serve",
-    "start:prod": "npm run prebuild && ng serve --configuration=serve-production",
-    "start-local": "npm run prebuild && ng serve --proxy-config local.proxy.conf.js --live-reload",
-    "start-local:aot": " npm run prebuild && ng serve --proxy-config local.proxy.conf.js --live-reload  --host=0.0.0.0 --configuration production --aot=true",
-    "dev": "npm run prebuild && ng serve --proxy-config local.proxy.conf.js --live-reload --host=0.0.0.0",
-    "dev-idp": "npm run prebuild && ng serve --proxy-config local.proxy.conf.js --live-reload --host=0.0.0.0 --disable-host-check  -c idp",
-    "dev:aot": "npm run prebuild && ng serve --proxy-config local.proxy.conf.js --live-reload  --host=0.0.0.0 --configuration production --aot=true",
+    "start": "npm run prebuild && ng serve --proxy-config local.proxy.conf.js --live-reload --host=0.0.0.0",
```
Comment (sergeysenja1992) [inline package.json:28]: add start without theme generation

Comment (Kaskyi) [reply package.json:28]: add start:minimal

File: package.json (hunk 2 above)
Comment (Kaskyi) [reply package.json:32]: done

**Review by zverbeta — APPROVED** [review]

**Review by sergeysenja1992 — APPROVED** [review]

**Pull request merged**